.pylintdict

```diff
@@ -186,6 +188,7 @@ rz
 sahar
 scipy
 sdp
+semideterministic
```

Comment:
I think the original source should be `semi-deterministic` with a hypen. In which case no custom word would be needed here I would think.